test/test_managercli.py

```diff
@@ -341,6 +350,11 @@ def setUp(self):
         argv_patcher = patch.object(sys, 'argv', ['subscription-manager', 'register'])
         argv_patcher.start()
         self.addCleanup(argv_patcher.stop)
+        intentfiles.USER_INTENT = self.write_tempfile("{}").name
+
```

Comment:
Please, can you add here some tests that `/etc/rhsm/intent/intent.json` was created and it was created properly?

Reply:
I've added a unit test for the service-level command, but it's near impossible to do the same for the register command, as the whole of `_do_command()` is mocked out.